.github/workflows/i18n.yml

```diff
@@ -10,6 +10,8 @@ jobs:
   locale-updater:
     name: Locale updater
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
```

Comment:
We tried this before on Jazzband to no avail, so I'm playing around with permissions to see if it works. Might need to add content: write, but I can't find default perms on gitHub docs

ref https://github.com/peter-evans/create-pull-request#action-inputs

Reply:
Oh, did you check this, the workflow permission's [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions)? 

```
All inputs are optional. If not set, sensible defaults will be used.
```

Since it is a minor change, we might work with this current implementation :) 

Ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions